src/components/atoms/Platform.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="platform" />
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from "vue-property-decorator";
+import { Platform as PlatformDomain } from "@/domain/platform";
+
+@Component
+export default class Platform extends Vue {
+  @Prop() private platform!: PlatformDomain;
+}
+</script>
+
+<style scoped lang="scss">
+@import "src/variables";
```

Comment:
We are gonna use those variables elsewhere I think? We should use https://css-tricks.com/how-to-import-a-sass-file-into-every-vue-component-in-an-app/